app/assets/javascripts/application.js

```diff
@@ -8,3 +8,90 @@ if (window.console && window.console.info) {
 $(document).ready(function () {
   window.GOVUKFrontend.initAll()
 })
+
+function DateTime () {
+  function getDaySuffix (a) {
+    var b = '' + a,
+      c = b.length,
+      d = parseInt(b.substring(c - 2, c - 1)),
+      e = parseInt(b.substring(c - 1))
```

Comment:
We should probably be kind to our future selves and give these variables [meaningful names](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c).